docs/index.rst

```diff
@@ -28,6 +28,8 @@ The ``brickschema`` package requires Python >= 3.6. It can be installed with ``p
 
    quickstart
    orm
+   validate
+   brick_validate
```

Comment:
Can this go under a "scripts" or "tools" section? `brick_validate` won't be the only tool we associate with this repo so it is worth doing the organization now

Reply:
On second thought, I think it's OK for the document to stay here while I have moved the brick_validate script to bin.

Reply:
It makes the tool discoverable showing up on the top page.